src/main/java/core/basesyntax/strategy/impl/SupplyOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.